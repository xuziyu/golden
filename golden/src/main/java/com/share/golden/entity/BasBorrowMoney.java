package com.share.golden.entity;

public class BasBorrowMoney {
    private String borrowId;

    private String borrowSerial;

    private String userId;

    private String borrowCompany;

    private String borrowMoney;

    private String borrowRate;

    private String timeLimit;

    private String borrowLimit;

    private String borrowState;

    private String borrowRecommend;

    private String borrowType;

    private String borrowRemark1;

    private String borrowRemark2;

    public String getBorrowId() {
        return borrowId;
    }

    public void setBorrowId(String borrowId) {
        this.borrowId = borrowId == null ? null : borrowId.trim();
    }

    public String getBorrowSerial() {
        return borrowSerial;
    }

    public void setBorrowSerial(String borrowSerial) {
        this.borrowSerial = borrowSerial == null ? null : borrowSerial.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getBorrowCompany() {
        return borrowCompany;
    }

    public void setBorrowCompany(String borrowCompany) {
        this.borrowCompany = borrowCompany == null ? null : borrowCompany.trim();
    }

    public String getBorrowMoney() {
        return borrowMoney;
    }

    public void setBorrowMoney(String borrowMoney) {
        this.borrowMoney = borrowMoney == null ? null : borrowMoney.trim();
    }

    public String getBorrowRate() {
        return borrowRate;
    }

    public void setBorrowRate(String borrowRate) {
        this.borrowRate = borrowRate == null ? null : borrowRate.trim();
    }

    public String getTimeLimit() {
        return timeLimit;
    }

    public void setTimeLimit(String timeLimit) {
        this.timeLimit = timeLimit == null ? null : timeLimit.trim();
    }

    public String getBorrowLimit() {
        return borrowLimit;
    }

    public void setBorrowLimit(String borrowLimit) {
        this.borrowLimit = borrowLimit == null ? null : borrowLimit.trim();
    }

    public String getBorrowState() {
        return borrowState;
    }

    public void setBorrowState(String borrowState) {
        this.borrowState = borrowState == null ? null : borrowState.trim();
    }

    public String getBorrowRecommend() {
        return borrowRecommend;
    }

    public void setBorrowRecommend(String borrowRecommend) {
        this.borrowRecommend = borrowRecommend == null ? null : borrowRecommend.trim();
    }

    public String getBorrowType() {
        return borrowType;
    }

    public void setBorrowType(String borrowType) {
        this.borrowType = borrowType == null ? null : borrowType.trim();
    }

    public String getBorrowRemark1() {
        return borrowRemark1;
    }

    public void setBorrowRemark1(String borrowRemark1) {
        this.borrowRemark1 = borrowRemark1 == null ? null : borrowRemark1.trim();
    }

    public String getBorrowRemark2() {
        return borrowRemark2;
    }

    public void setBorrowRemark2(String borrowRemark2) {
        this.borrowRemark2 = borrowRemark2 == null ? null : borrowRemark2.trim();
    }
}