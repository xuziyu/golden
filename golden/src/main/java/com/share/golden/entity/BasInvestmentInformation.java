package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasInvestmentInformation {
    private String investorId;

    private String userId;

    private String tenderId;

    private BigDecimal investorMoney;

    private String investorStatus;

    private String investorStyle;

    private String brrowStatus;

    private String investorRest;

    private String profitModel;

    private BigDecimal profitMoney;

    private Date investorDate;

    private String replayDate;

    private String investorType;

    public String getInvestorId() {
        return investorId;
    }

    public void setInvestorId(String investorId) {
        this.investorId = investorId == null ? null : investorId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getTenderId() {
        return tenderId;
    }

    public void setTenderId(String tenderId) {
        this.tenderId = tenderId == null ? null : tenderId.trim();
    }

    public BigDecimal getInvestorMoney() {
        return investorMoney;
    }

    public void setInvestorMoney(BigDecimal investorMoney) {
        this.investorMoney = investorMoney;
    }

    public String getInvestorStatus() {
        return investorStatus;
    }

    public void setInvestorStatus(String investorStatus) {
        this.investorStatus = investorStatus == null ? null : investorStatus.trim();
    }

    public String getInvestorStyle() {
        return investorStyle;
    }

    public void setInvestorStyle(String investorStyle) {
        this.investorStyle = investorStyle == null ? null : investorStyle.trim();
    }

    public String getBrrowStatus() {
        return brrowStatus;
    }

    public void setBrrowStatus(String brrowStatus) {
        this.brrowStatus = brrowStatus == null ? null : brrowStatus.trim();
    }

    public String getInvestorRest() {
        return investorRest;
    }

    public void setInvestorRest(String investorRest) {
        this.investorRest = investorRest == null ? null : investorRest.trim();
    }

    public String getProfitModel() {
        return profitModel;
    }

    public void setProfitModel(String profitModel) {
        this.profitModel = profitModel == null ? null : profitModel.trim();
    }

    public BigDecimal getProfitMoney() {
        return profitMoney;
    }

    public void setProfitMoney(BigDecimal profitMoney) {
        this.profitMoney = profitMoney;
    }

    public Date getInvestorDate() {
        return investorDate;
    }

    public void setInvestorDate(Date investorDate) {
        this.investorDate = investorDate;
    }

    public String getReplayDate() {
        return replayDate;
    }

    public void setReplayDate(String replayDate) {
        this.replayDate = replayDate == null ? null : replayDate.trim();
    }

    public String getInvestorType() {
        return investorType;
    }

    public void setInvestorType(String investorType) {
        this.investorType = investorType == null ? null : investorType.trim();
    }
}