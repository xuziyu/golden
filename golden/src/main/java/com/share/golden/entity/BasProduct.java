package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasProduct {
    private String productId;

    private BigDecimal productMoney;

    private String productIncome;

    private Date productTime;

    private String productType;

    private String productWay;

    private Date pcount;

    private String productProgress;

    private String productSaveway;

    private String productRateben;

    private Date productPublishTime;

    private String productName;

    private BigDecimal productTotalMoney;

    private String investmentScope;

    private String useOfProceeds;

    private String productState;

    private String productPicture;

    private String produceId;

    private String productDesc;

    private String productSafe;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId == null ? null : productId.trim();
    }

    public BigDecimal getProductMoney() {
        return productMoney;
    }

    public void setProductMoney(BigDecimal productMoney) {
        this.productMoney = productMoney;
    }

    public String getProductIncome() {
        return productIncome;
    }

    public void setProductIncome(String productIncome) {
        this.productIncome = productIncome == null ? null : productIncome.trim();
    }

    public Date getProductTime() {
        return productTime;
    }

    public void setProductTime(Date productTime) {
        this.productTime = productTime;
    }

    public String getProductType() {
        return productType;
    }

    public void setProductType(String productType) {
        this.productType = productType == null ? null : productType.trim();
    }

    public String getProductWay() {
        return productWay;
    }

    public void setProductWay(String productWay) {
        this.productWay = productWay == null ? null : productWay.trim();
    }

    public Date getPcount() {
        return pcount;
    }

    public void setPcount(Date pcount) {
        this.pcount = pcount;
    }

    public String getProductProgress() {
        return productProgress;
    }

    public void setProductProgress(String productProgress) {
        this.productProgress = productProgress == null ? null : productProgress.trim();
    }

    public String getProductSaveway() {
        return productSaveway;
    }

    public void setProductSaveway(String productSaveway) {
        this.productSaveway = productSaveway == null ? null : productSaveway.trim();
    }

    public String getProductRateben() {
        return productRateben;
    }

    public void setProductRateben(String productRateben) {
        this.productRateben = productRateben == null ? null : productRateben.trim();
    }

    public Date getProductPublishTime() {
        return productPublishTime;
    }

    public void setProductPublishTime(Date productPublishTime) {
        this.productPublishTime = productPublishTime;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName == null ? null : productName.trim();
    }

    public BigDecimal getProductTotalMoney() {
        return productTotalMoney;
    }

    public void setProductTotalMoney(BigDecimal productTotalMoney) {
        this.productTotalMoney = productTotalMoney;
    }

    public String getInvestmentScope() {
        return investmentScope;
    }

    public void setInvestmentScope(String investmentScope) {
        this.investmentScope = investmentScope == null ? null : investmentScope.trim();
    }

    public String getUseOfProceeds() {
        return useOfProceeds;
    }

    public void setUseOfProceeds(String useOfProceeds) {
        this.useOfProceeds = useOfProceeds == null ? null : useOfProceeds.trim();
    }

    public String getProductState() {
        return productState;
    }

    public void setProductState(String productState) {
        this.productState = productState == null ? null : productState.trim();
    }

    public String getProductPicture() {
        return productPicture;
    }

    public void setProductPicture(String productPicture) {
        this.productPicture = productPicture == null ? null : productPicture.trim();
    }

    public String getProduceId() {
        return produceId;
    }

    public void setProduceId(String produceId) {
        this.produceId = produceId == null ? null : produceId.trim();
    }

    public String getProductDesc() {
        return productDesc;
    }

    public void setProductDesc(String productDesc) {
        this.productDesc = productDesc == null ? null : productDesc.trim();
    }

    public String getProductSafe() {
        return productSafe;
    }

    public void setProductSafe(String productSafe) {
        this.productSafe = productSafe == null ? null : productSafe.trim();
    }
}