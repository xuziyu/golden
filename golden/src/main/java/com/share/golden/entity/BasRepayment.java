package com.share.golden.entity;

import java.util.Date;

public class BasRepayment {
    private String repayId;

    private String repayCode;

    private Date repayDate;

    private String repayState;

    private String repayTime;

    private String repayRemark1;

    private String repayRemark2;

    public String getRepayId() {
        return repayId;
    }

    public void setRepayId(String repayId) {
        this.repayId = repayId == null ? null : repayId.trim();
    }

    public String getRepayCode() {
        return repayCode;
    }

    public void setRepayCode(String repayCode) {
        this.repayCode = repayCode == null ? null : repayCode.trim();
    }

    public Date getRepayDate() {
        return repayDate;
    }

    public void setRepayDate(Date repayDate) {
        this.repayDate = repayDate;
    }

    public String getRepayState() {
        return repayState;
    }

    public void setRepayState(String repayState) {
        this.repayState = repayState == null ? null : repayState.trim();
    }

    public String getRepayTime() {
        return repayTime;
    }

    public void setRepayTime(String repayTime) {
        this.repayTime = repayTime == null ? null : repayTime.trim();
    }

    public String getRepayRemark1() {
        return repayRemark1;
    }

    public void setRepayRemark1(String repayRemark1) {
        this.repayRemark1 = repayRemark1 == null ? null : repayRemark1.trim();
    }

    public String getRepayRemark2() {
        return repayRemark2;
    }

    public void setRepayRemark2(String repayRemark2) {
        this.repayRemark2 = repayRemark2 == null ? null : repayRemark2.trim();
    }
}