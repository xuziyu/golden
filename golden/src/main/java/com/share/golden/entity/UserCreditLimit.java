package com.share.golden.entity;

public class UserCreditLimit {
	private String limitCode;

	private String userId;

	private String creditLimit;

	private String creditGrade;

	public String getLimitCode() {
		return limitCode;
	}

	public void setLimitCode(String limitCode) {
		this.limitCode = limitCode == null ? null : limitCode.trim();
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId == null ? null : userId.trim();
	}

	public String getCreditLimit() {
		return creditLimit;
	}

	public void setCreditLimit(String creditLimit) {
		this.creditLimit = creditLimit == null ? null : creditLimit.trim();
	}

	public String getCreditGrade() {
		return creditGrade;
	}

	public void setCreditGrade(String creditGrade) {
		this.creditGrade = creditGrade == null ? null : creditGrade.trim();
	}
}