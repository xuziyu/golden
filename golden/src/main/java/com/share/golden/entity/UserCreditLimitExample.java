package com.share.golden.entity;

import java.util.ArrayList;
import java.util.List;

public class UserCreditLimitExample {
    protected String orderByClause;

    protected boolean distinct;

    protected List<Criteria> oredCriteria;

    public UserCreditLimitExample() {
        oredCriteria = new ArrayList<Criteria>();
    }

    public void setOrderByClause(String orderByClause) {
        this.orderByClause = orderByClause;
    }

    public String getOrderByClause() {
        return orderByClause;
    }

    public void setDistinct(boolean distinct) {
        this.distinct = distinct;
    }

    public boolean isDistinct() {
        return distinct;
    }

    public List<Criteria> getOredCriteria() {
        return oredCriteria;
    }

    public void or(Criteria criteria) {
        oredCriteria.add(criteria);
    }

    public Criteria or() {
        Criteria criteria = createCriteriaInternal();
        oredCriteria.add(criteria);
        return criteria;
    }

    public Criteria createCriteria() {
        Criteria criteria = createCriteriaInternal();
        if (oredCriteria.size() == 0) {
            oredCriteria.add(criteria);
        }
        return criteria;
    }

    protected Criteria createCriteriaInternal() {
        Criteria criteria = new Criteria();
        return criteria;
    }

    public void clear() {
        oredCriteria.clear();
        orderByClause = null;
        distinct = false;
    }

    protected abstract static class GeneratedCriteria {
        protected List<Criterion> criteria;

        protected GeneratedCriteria() {
            super();
            criteria = new ArrayList<Criterion>();
        }

        public boolean isValid() {
            return criteria.size() > 0;
        }

        public List<Criterion> getAllCriteria() {
            return criteria;
        }

        public List<Criterion> getCriteria() {
            return criteria;
        }

        protected void addCriterion(String condition) {
            if (condition == null) {
                throw new RuntimeException("Value for condition cannot be null");
            }
            criteria.add(new Criterion(condition));
        }

        protected void addCriterion(String condition, Object value, String property) {
            if (value == null) {
                throw new RuntimeException("Value for " + property + " cannot be null");
            }
            criteria.add(new Criterion(condition, value));
        }

        protected void addCriterion(String condition, Object value1, Object value2, String property) {
            if (value1 == null || value2 == null) {
                throw new RuntimeException("Between values for " + property + " cannot be null");
            }
            criteria.add(new Criterion(condition, value1, value2));
        }

        public Criteria andLimitCodeIsNull() {
            addCriterion("\"LIMIT_CODE\" is null");
            return (Criteria) this;
        }

        public Criteria andLimitCodeIsNotNull() {
            addCriterion("\"LIMIT_CODE\" is not null");
            return (Criteria) this;
        }

        public Criteria andLimitCodeEqualTo(String value) {
            addCriterion("\"LIMIT_CODE\" =", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeNotEqualTo(String value) {
            addCriterion("\"LIMIT_CODE\" <>", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeGreaterThan(String value) {
            addCriterion("\"LIMIT_CODE\" >", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeGreaterThanOrEqualTo(String value) {
            addCriterion("\"LIMIT_CODE\" >=", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeLessThan(String value) {
            addCriterion("\"LIMIT_CODE\" <", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeLessThanOrEqualTo(String value) {
            addCriterion("\"LIMIT_CODE\" <=", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeLike(String value) {
            addCriterion("\"LIMIT_CODE\" like", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeNotLike(String value) {
            addCriterion("\"LIMIT_CODE\" not like", value, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeIn(List<String> values) {
            addCriterion("\"LIMIT_CODE\" in", values, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeNotIn(List<String> values) {
            addCriterion("\"LIMIT_CODE\" not in", values, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeBetween(String value1, String value2) {
            addCriterion("\"LIMIT_CODE\" between", value1, value2, "limitCode");
            return (Criteria) this;
        }

        public Criteria andLimitCodeNotBetween(String value1, String value2) {
            addCriterion("\"LIMIT_CODE\" not between", value1, value2, "limitCode");
            return (Criteria) this;
        }

        public Criteria andUserIdIsNull() {
            addCriterion("\"USER_ID\" is null");
            return (Criteria) this;
        }

        public Criteria andUserIdIsNotNull() {
            addCriterion("\"USER_ID\" is not null");
            return (Criteria) this;
        }

        public Criteria andUserIdEqualTo(String value) {
            addCriterion("\"USER_ID\" =", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdNotEqualTo(String value) {
            addCriterion("\"USER_ID\" <>", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdGreaterThan(String value) {
            addCriterion("\"USER_ID\" >", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdGreaterThanOrEqualTo(String value) {
            addCriterion("\"USER_ID\" >=", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdLessThan(String value) {
            addCriterion("\"USER_ID\" <", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdLessThanOrEqualTo(String value) {
            addCriterion("\"USER_ID\" <=", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdLike(String value) {
            addCriterion("\"USER_ID\" like", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdNotLike(String value) {
            addCriterion("\"USER_ID\" not like", value, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdIn(List<String> values) {
            addCriterion("\"USER_ID\" in", values, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdNotIn(List<String> values) {
            addCriterion("\"USER_ID\" not in", values, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdBetween(String value1, String value2) {
            addCriterion("\"USER_ID\" between", value1, value2, "userId");
            return (Criteria) this;
        }

        public Criteria andUserIdNotBetween(String value1, String value2) {
            addCriterion("\"USER_ID\" not between", value1, value2, "userId");
            return (Criteria) this;
        }

        public Criteria andCreditLimitIsNull() {
            addCriterion("\"CREDIT_LIMIT\" is null");
            return (Criteria) this;
        }

        public Criteria andCreditLimitIsNotNull() {
            addCriterion("\"CREDIT_LIMIT\" is not null");
            return (Criteria) this;
        }

        public Criteria andCreditLimitEqualTo(String value) {
            addCriterion("\"CREDIT_LIMIT\" =", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitNotEqualTo(String value) {
            addCriterion("\"CREDIT_LIMIT\" <>", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitGreaterThan(String value) {
            addCriterion("\"CREDIT_LIMIT\" >", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitGreaterThanOrEqualTo(String value) {
            addCriterion("\"CREDIT_LIMIT\" >=", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitLessThan(String value) {
            addCriterion("\"CREDIT_LIMIT\" <", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitLessThanOrEqualTo(String value) {
            addCriterion("\"CREDIT_LIMIT\" <=", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitLike(String value) {
            addCriterion("\"CREDIT_LIMIT\" like", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitNotLike(String value) {
            addCriterion("\"CREDIT_LIMIT\" not like", value, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitIn(List<String> values) {
            addCriterion("\"CREDIT_LIMIT\" in", values, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitNotIn(List<String> values) {
            addCriterion("\"CREDIT_LIMIT\" not in", values, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitBetween(String value1, String value2) {
            addCriterion("\"CREDIT_LIMIT\" between", value1, value2, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditLimitNotBetween(String value1, String value2) {
            addCriterion("\"CREDIT_LIMIT\" not between", value1, value2, "creditLimit");
            return (Criteria) this;
        }

        public Criteria andCreditGradeIsNull() {
            addCriterion("\"CREDIT_GRADE\" is null");
            return (Criteria) this;
        }

        public Criteria andCreditGradeIsNotNull() {
            addCriterion("\"CREDIT_GRADE\" is not null");
            return (Criteria) this;
        }

        public Criteria andCreditGradeEqualTo(String value) {
            addCriterion("\"CREDIT_GRADE\" =", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeNotEqualTo(String value) {
            addCriterion("\"CREDIT_GRADE\" <>", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeGreaterThan(String value) {
            addCriterion("\"CREDIT_GRADE\" >", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeGreaterThanOrEqualTo(String value) {
            addCriterion("\"CREDIT_GRADE\" >=", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeLessThan(String value) {
            addCriterion("\"CREDIT_GRADE\" <", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeLessThanOrEqualTo(String value) {
            addCriterion("\"CREDIT_GRADE\" <=", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeLike(String value) {
            addCriterion("\"CREDIT_GRADE\" like", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeNotLike(String value) {
            addCriterion("\"CREDIT_GRADE\" not like", value, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeIn(List<String> values) {
            addCriterion("\"CREDIT_GRADE\" in", values, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeNotIn(List<String> values) {
            addCriterion("\"CREDIT_GRADE\" not in", values, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeBetween(String value1, String value2) {
            addCriterion("\"CREDIT_GRADE\" between", value1, value2, "creditGrade");
            return (Criteria) this;
        }

        public Criteria andCreditGradeNotBetween(String value1, String value2) {
            addCriterion("\"CREDIT_GRADE\" not between", value1, value2, "creditGrade");
            return (Criteria) this;
        }
    }

    public static class Criteria extends GeneratedCriteria {

        protected Criteria() {
            super();
        }
    }

    public static class Criterion {
        private String condition;

        private Object value;

        private Object secondValue;

        private boolean noValue;

        private boolean singleValue;

        private boolean betweenValue;

        private boolean listValue;

        private String typeHandler;

        public String getCondition() {
            return condition;
        }

        public Object getValue() {
            return value;
        }

        public Object getSecondValue() {
            return secondValue;
        }

        public boolean isNoValue() {
            return noValue;
        }

        public boolean isSingleValue() {
            return singleValue;
        }

        public boolean isBetweenValue() {
            return betweenValue;
        }

        public boolean isListValue() {
            return listValue;
        }

        public String getTypeHandler() {
            return typeHandler;
        }

        protected Criterion(String condition) {
            super();
            this.condition = condition;
            this.typeHandler = null;
            this.noValue = true;
        }

        protected Criterion(String condition, Object value, String typeHandler) {
            super();
            this.condition = condition;
            this.value = value;
            this.typeHandler = typeHandler;
            if (value instanceof List<?>) {
                this.listValue = true;
            } else {
                this.singleValue = true;
            }
        }

        protected Criterion(String condition, Object value) {
            this(condition, value, null);
        }

        protected Criterion(String condition, Object value, Object secondValue, String typeHandler) {
            super();
            this.condition = condition;
            this.value = value;
            this.secondValue = secondValue;
            this.typeHandler = typeHandler;
            this.betweenValue = true;
        }

        protected Criterion(String condition, Object value, Object secondValue) {
            this(condition, value, secondValue, null);
        }
    }
}