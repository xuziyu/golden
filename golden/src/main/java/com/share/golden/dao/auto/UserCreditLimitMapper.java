package com.share.golden.dao.auto;

import com.share.golden.entity.UserCreditLimit;
import com.share.golden.entity.UserCreditLimitExample;
import java.util.List;
import org.apache.ibatis.annotations.Param;

public interface UserCreditLimitMapper {
    long countByExample(UserCreditLimitExample example);

    int deleteByExample(UserCreditLimitExample example);

    int deleteByPrimaryKey(String limitCode);

    int insert(UserCreditLimit record);

    int insertSelective(UserCreditLimit record);

    List<UserCreditLimit> selectByExample(UserCreditLimitExample example);

    UserCreditLimit selectByPrimaryKey(String limitCode);

    int updateByExampleSelective(@Param("record") UserCreditLimit record, @Param("example") UserCreditLimitExample example);

    int updateByExample(@Param("record") UserCreditLimit record, @Param("example") UserCreditLimitExample example);

    int updateByPrimaryKeySelective(UserCreditLimit record);

    int updateByPrimaryKey(UserCreditLimit record);
}