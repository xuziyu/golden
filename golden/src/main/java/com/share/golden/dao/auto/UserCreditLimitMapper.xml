<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.share.golden.dao.auto.UserCreditLimitMapper">
  <resultMap id="BaseResultMap" type="com.share.golden.entity.UserCreditLimit">
    <id column="LIMIT_CODE" jdbcType="VARCHAR" property="limitCode" />
    <result column="USER_ID" jdbcType="VARCHAR" property="userId" />
    <result column="CREDIT_LIMIT" jdbcType="VARCHAR" property="creditLimit" />
    <result column="CREDIT_GRADE" jdbcType="VARCHAR" property="creditGrade" />
  </resultMap>
  <sql id="Example_Where_Clause">
    <where>
      <foreach collection="oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Update_By_Example_Where_Clause">
    <where>
      <foreach collection="example.oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Base_Column_List">
    "LIMIT_CODE", "USER_ID", "CREDIT_LIMIT", "CREDIT_GRADE"
  </sql>
  <select id="selectByExample" parameterType="com.share.golden.entity.UserCreditLimitExample" resultMap="BaseResultMap">
    select
    <if test="distinct">
      distinct
    </if>
    <include refid="Base_Column_List" />
    from USER_CREDIT_LIMIT
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
    <if test="orderByClause != null">
      order by ${orderByClause}
    </if>
  </select>
  <select id="selectByPrimaryKey" parameterType="java.lang.String" resultMap="BaseResultMap">
    select 
    <include refid="Base_Column_List" />
    from USER_CREDIT_LIMIT
    where "LIMIT_CODE" = #{limitCode,jdbcType=VARCHAR}
  </select>
  <delete id="deleteByPrimaryKey" parameterType="java.lang.String">
    delete from USER_CREDIT_LIMIT
    where "LIMIT_CODE" = #{limitCode,jdbcType=VARCHAR}
  </delete>
  <delete id="deleteByExample" parameterType="com.share.golden.entity.UserCreditLimitExample">
    delete from USER_CREDIT_LIMIT
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </delete>
  <insert id="insert" parameterType="com.share.golden.entity.UserCreditLimit">
    insert into USER_CREDIT_LIMIT ("LIMIT_CODE", "USER_ID", "CREDIT_LIMIT", 
      "CREDIT_GRADE")
    values (#{limitCode,jdbcType=VARCHAR}, #{userId,jdbcType=VARCHAR}, #{creditLimit,jdbcType=VARCHAR}, 
      #{creditGrade,jdbcType=VARCHAR})
  </insert>
  <insert id="insertSelective" parameterType="com.share.golden.entity.UserCreditLimit">
    insert into USER_CREDIT_LIMIT
    <trim prefix="(" suffix=")" suffixOverrides=",">
      <if test="limitCode != null">
        "LIMIT_CODE",
      </if>
      <if test="userId != null">
        "USER_ID",
      </if>
      <if test="creditLimit != null">
        "CREDIT_LIMIT",
      </if>
      <if test="creditGrade != null">
        "CREDIT_GRADE",
      </if>
    </trim>
    <trim prefix="values (" suffix=")" suffixOverrides=",">
      <if test="limitCode != null">
        #{limitCode,jdbcType=VARCHAR},
      </if>
      <if test="userId != null">
        #{userId,jdbcType=VARCHAR},
      </if>
      <if test="creditLimit != null">
        #{creditLimit,jdbcType=VARCHAR},
      </if>
      <if test="creditGrade != null">
        #{creditGrade,jdbcType=VARCHAR},
      </if>
    </trim>
  </insert>
  <select id="countByExample" parameterType="com.share.golden.entity.UserCreditLimitExample" resultType="java.lang.Long">
    select count(*) from USER_CREDIT_LIMIT
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </select>
  <update id="updateByExampleSelective" parameterType="map">
    update USER_CREDIT_LIMIT
    <set>
      <if test="record.limitCode != null">
        "LIMIT_CODE" = #{record.limitCode,jdbcType=VARCHAR},
      </if>
      <if test="record.userId != null">
        "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      </if>
      <if test="record.creditLimit != null">
        "CREDIT_LIMIT" = #{record.creditLimit,jdbcType=VARCHAR},
      </if>
      <if test="record.creditGrade != null">
        "CREDIT_GRADE" = #{record.creditGrade,jdbcType=VARCHAR},
      </if>
    </set>
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByExample" parameterType="map">
    update USER_CREDIT_LIMIT
    set "LIMIT_CODE" = #{record.limitCode,jdbcType=VARCHAR},
      "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      "CREDIT_LIMIT" = #{record.creditLimit,jdbcType=VARCHAR},
      "CREDIT_GRADE" = #{record.creditGrade,jdbcType=VARCHAR}
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByPrimaryKeySelective" parameterType="com.share.golden.entity.UserCreditLimit">
    update USER_CREDIT_LIMIT
    <set>
      <if test="userId != null">
        "USER_ID" = #{userId,jdbcType=VARCHAR},
      </if>
      <if test="creditLimit != null">
        "CREDIT_LIMIT" = #{creditLimit,jdbcType=VARCHAR},
      </if>
      <if test="creditGrade != null">
        "CREDIT_GRADE" = #{creditGrade,jdbcType=VARCHAR},
      </if>
    </set>
    where "LIMIT_CODE" = #{limitCode,jdbcType=VARCHAR}
  </update>
  <update id="updateByPrimaryKey" parameterType="com.share.golden.entity.UserCreditLimit">
    update USER_CREDIT_LIMIT
    set "USER_ID" = #{userId,jdbcType=VARCHAR},
      "CREDIT_LIMIT" = #{creditLimit,jdbcType=VARCHAR},
      "CREDIT_GRADE" = #{creditGrade,jdbcType=VARCHAR}
    where "LIMIT_CODE" = #{limitCode,jdbcType=VARCHAR}
  </update>
</mapper>